lib/services/event_service.ex

```diff
@@ -1,15 +1,20 @@
 defmodule Aprb.Service.EventService do
-  alias Aprb.{Repo, Topic}
+  alias Aprb.{Repo, Topic, Service.SummaryService}
+
   def receive_event(event, topic) do
-    proccessed_message = process_event(event, topic)
+    proccessed_message = process_event(decode_event(event), topic)
```

Comment:
Is it more idiomatic to use the `|>` syntax here? I don't really know but I rarely see nested function calls in Elixir.